VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/data/Credentials.java

```diff
@@ -1,11 +1,26 @@
 package org.edx.mobile.test.feature.data;
 
+import android.support.annotation.Nullable;
+import org.edx.mobile.test.feature.config.E2EConfig;
+import org.edx.mobile.util.ResourceUtil;
+
+import java.util.UUID;
+
 public class Credentials {
     public final String email;
     public final String password;
+    @Nullable  public String username;
 
     public Credentials(String email, String password) {
         this.email = email;
         this.password = password;
     }
+
+    public static Credentials forRegistration() {
```

Comment:
There are other cases where we will need fresh credentails, so instead of calling this ``forRegistration`` lets just call it ``freshCredentials``?

(In particular, we should change the other tests to generate a credential at the start of all the tests that we can then use for all non registration tests).